packages/commonwealth/client/scripts/views/pages/create_community/index.tsx

```diff
@@ -106,7 +109,7 @@ const CreateCommunity = (props: CreateCommunityProps) => {
     } else {
       setCurrentForm(getFormType(type));
     }
-  }, [type]);
+  }, [type, navigate]);
```

Comment:
Suggestion (unrelated to pr): 

since the `navigate` from `useCommonNavigate` doesn't get updated, maybe we can memorize it. Right now this effect seems to run 6-7 times, I think due to the navigate cb reference getting updated

cc: @masvelio @Miaplacidus

Reply:
@Miaplacidus added `navigate` to the dependency array here to satisfy eslint which is good. @mzparacha your point about memoization is good though, but this is out of scope for this PR. We can investigate it in different ticket

Reply:
@mzparacha Fair point! Will you be creating a ticket around that?